### DISCUSSION TITLE
[Help Wanted] Cannot rename 1st spreadsheet tab

### DISCUSSION THREAD

**Discussion by MorgothSauron** [standardnotes/forum#2835] [General]
**Describe the issue**
I'm unable to rename the 1st tab in a spreadsheet. The tab will quickly switch to the edit mode before reverting to the normal state.

The tab can be renamed if it is moved to the right of an other tab. The "new" first tab cannot be renamed.

**To Reproduce**
1. Create a new spreadsheet
2. Double click on the first tab
3. Unable to rename the tab

**Expected behavior**
Be able to rename a tab

**Desktop (please complete the following information):**
 - OS: 5.18.10-gentoo-x86_64
 - Browser Firefox
 - Version 102.0

**Smartphone (please complete the following information):**
 - Device: xiaomi Redmi Note 8 Pro
 - OS: Android 12.1

**Additional context**
- I have two different spreadsheet that shows this behavior. Renaming other tabs than the first one works.
- I experience the same behavior in the desktop app, the web app and on android

**Comment by MorgothSauron** [comment]
Here's a recording of the Desktop App
https://user-images.githubusercontent.com/6804228/178556006-133e848d-cfb3-4edd-8c4d-e6ee802639e9.mp4

![standardnotes](https://user-images.githubusercontent.com/6804228/178557399-81295ce0-7db5-444f-9327-b7e7c4b6fefa.gif)

**Comment by myreli** [comment]
Hi, @MorgothSauron are you still facing this issue? 

I wasn't able to reproduce it in the latest version, both on Android and the Desktop app.

**Comment by MorgothSauron** [comment]
I tried again with Desktop app v3.149.5 and I notice the same behavior: I double-click on the first tab to rename it but it quickly "exits" the edit mode. No issue with the other tabs.

https://user-images.githubusercontent.com/6804228/220979255-792c2c6c-54e9-46e5-a918-edd0c621bd40.mp4

**Comment by myreli** [comment]
I see, thank you for testing it again @MorgothSauron! I will transfer this to the new discussions and label it as a bug, so we can try and find out in which conditions this happens.
